### PR TITLE
[now-cli] Fix chalk template for legacy

### LINKED ISSUE #3755: [now-cli] Error when deploying now v1 project: Chalk template literal is missing 1 closing bracket (`}`)
After upgrading to `now@17.0.1` I'm getting the following error: `Error: Chalk template literal is missing 1 closing bracket (`}`)` during the deployment process.

Here is the complete console output:
```
Now CLI 17.0.1
┌──────────────────────────────────────────────────────────────────────────────────────────────────────────┐
│ WARN! You are using an old version of the Now Platform. More: https://zeit.co/guides/migrate-to-zeit-now │
└──────────────────────────────────────────────────────────────────────────────────────────────────────────┘
> Deploying ~/tmp under #######
> Using project #######
> https://#######.now.sh [v1] [2s]
> Build completed
Error! An unexpected error occurred!
  Error: Chalk template literal is missing 1 closing bracket (`}`)
    at e.exports (/usr/local/lib/node_modules/now/dist/index.js:2:1500109)
    at chalkTag (/usr/local/lib/node_modules/now/dist/index.js:2:4107323)
    at Object.Chalk.t.template [as default] (/usr/local/lib/node_modules/now/dist/index.js:2:4105020)
    at /usr/local/lib/node_modules/now/dist/index.js:2:4024265
    at processTicksAndRejections (internal/process/task_queues.js:94:5) Error: Chalk template literal is missing 1 closing bracket (`}`)
    at e.exports (/usr/local/lib/node_modules/now/dist/index.js:2:1500109)
    at chalkTag (/usr/local/lib/node_modules/now/dist/index.js:2:4107323)
    at Object.Chalk.t.template [as default] (/usr/local/lib/node_modules/now/dist/index.js:2:4105020)
    at /usr/local/lib/node_modules/now/dist/index.js:2:4024265
    at processTicksAndRejections (internal/process/task_queues.js:94:5)

Exited with code exit status 1
```

### PR DESCRIPTION
**Pull request by AndyBitz**
Fixes https://github.com/zeit/now/issues/3755

### CODE REVIEW

File: packages/now-cli/src/commands/deploy/legacy.ts [hunk 1]
```diff
@@ -852,7 +852,7 @@ async function sync({
     const { url } = now;
     const dcs =
       deploymentType !== 'static' && deployment.scale
-        ? chalk` ({bold ${Object.keys(deployment.scale).join(', ')})`
+        ? chalk(`${chalk.bold(Object.keys(deployment.scale).join(', '))}`)
```
Comment (TooTallNate) [inline packages/now-cli/src/commands/deploy/legacy.ts]:
```suggestion
        ? chalk` ({bold ${Object.keys(deployment.scale).join(', ')}})`
```

Comment (AndyBitz) [reply packages/now-cli/src/commands/deploy/legacy.ts]: Why keep the template function? I think it's just more confusing and error prone

Comment (TooTallNate) [reply packages/now-cli/src/commands/deploy/legacy.ts]: I don't think the outer `chalk()` function is necessary then, just use a JS string template and `chalk.bold()` inside.

Comment (TooTallNate) [reply packages/now-cli/src/commands/deploy/legacy.ts]: I think this matches the previous output:

```suggestion
        ? ` (${ chalk.bold(Object.keys(deployment.scale).join(', ')) })`
```

**Review by TooTallNate — APPROVED** [review]

**Review by styfle — APPROVED** [review]
Thanks!

**Pull request merged**